tools/setup_git_secrets.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
```

Comment:
Oh, I see. Bummer! I think that's a blocker; having to pay attention / do work to keep the secrets list in sync defeats the utility of having something check for secrets, to me. (It's the difference between one-time setup, where if I forget to install git-secrets some scripts break, and having to remember to re-run the installs when we add/change what patterns to looks for; the potential for silent failures.)

On the RDR, we got away with a simple `grep`. I'm inclined to stick with something like that. It looks like this https://github.com/vanderbilt/pmi-data/blob/master/ci/test.sh#L7 : 
```
echo "Grepping for checked-in credentials..."
set +e  # OK if grep does not find any matches.
KEY_FILES=`grep -ril "BEGIN PRIVATE KEY" . | grep -v $0 | grep -v oauth2client`
set -e
if [ "${KEY_FILES}" ]
then
  echo "No keys may be checked in, but found: $KEY_FILES"
  exit 1
fi
```